.github/workflows/ci.yml

```diff
@@ -92,3 +92,44 @@ jobs:
           DEFAULT_BRANCH: master
           # Not needed here as only one Linter is used.
           #GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  run-examples:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get update;
+          DEBIAN_FRONTEND=noninteractive sudo apt-get install --no-install-recommends -yq \
+            jq \
```

Comment:
```suggestion
```
This doesn't seem to actually be used.

Reply:
oh forgot to remove that one from my tests 👍